webapp/channels/src/components/widgets/inputs/check/check.scss

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+.check-input {
+    display: flex;
+    align-items: flex-start;
+    margin-top: 24px;
+    margin-bottom: 32px;
+
+    .text {
+        margin-left: 8px;
+        color: #3f4350;
```

Comment:
Can we use some other color for this? That's a portal color but isn't anywhere in mattermost-webapp yet. Since signup is something everyone goes through its a highly visible area where we want to be consistent with the rest of the product.

Reply:
Hey @annelieseklein could you help provide themable colors?

Reply:
atm, updated to `var(--center-channel-color-rgb)`

Reply:
another color for what exactly?